auter.yumdnfModule

```diff
@@ -113,6 +113,7 @@ function apply_updates() {
 
     local TRANSACTIONID=$(${PACKAGE_MANAGER} history info 2>&1 | grep "Transaction ID")
     logit "INFO: Updates complete (${PACKAGE_MANAGER} ${TRANSACTIONID}). You may need to reboot for some updates to take effect"
+    log_last_run
```

Comment:
That's actually deliberate. As a person running Auter, I care about the date that it last run successfully, because that's what I need to know for compliance.

If it exits without applying updates, I don't care what time it last ran because it's as if it didn't run at all - the system hasn't been updated. By only logging the time of successfully applying updates, we can simply check for that time and know that the system was updated then.

If we log the time for unsuccessful runs, we then need to check not only the time, but also try to determine the success status relating to that time.

I could move the logging into the quit function I guess and then log the time and exit code before dying if you think that would be more appropriate - then compliance can be checked by looking for the time and the exit code in that log message.